apps/src/aichat/views/modelCustomization/PublishNotes.tsx

```diff
@@ -64,7 +64,7 @@ const PublishNotes: React.FunctionComponent = () => {
                     readOnly={isDisabled(visibility)}
                   />
                 )}
-                {property !== 'exampleTopics' && (
+                {property !== 'exampleTopics' && property !== 'isPublished' && (
```

Comment:
Typescript did not like `!['exampleTopics', 'isPublished'].includes(property)` for some reason

Reply:
Nit: checking for these two properties here and in `ModelCardFields`. Maybe a helper function could be defined in `modelCustomiztion/utils`?

Reply:
Any more thoughts on customizing `PublishStatus` if `isPublished` is already set to `true` but user is just updating a model card field?

Reply:
And re: helper function, TS being annoying! Even if I move the exact same logic into a helper, I get this error:

```
ERROR in ./src/aichat/views/presentation/PresentationView.tsx:62:15
TS2322: Type 'string | boolean | string[]' is not assignable to type 'string | string[]'.
  Type 'boolean' is not assignable to type 'string | string[]'.
    60 |               title={label}
    61 |               titleIcon={iconName}
  > 62 |               expandedContent={modelCardInfo[property]}
       |               ^^^^^^^^^^^^^^^
    63 |               key={property}
    64 |             />
    65 |           );
```

Reply:
I appreciate you trying and sharing! I think it's fine as is - thanks!